docs/docs/mailer.md

```diff
@@ -0,0 +1,223 @@
+# Mailer
+
+Redwood provides a convenient-to-use mailer that you can reach for when you need to start sending emails to your users.
+
+## Architecture
+
+Redwood's mailer is comprised of handlers and renderers which each perform the core functionality of sending your emails out into the world and rendering your emails respectively. This is combined with just a few required files which define the necessary configuration.
+
+### Renderers
+
+A 'renderer' is responsible for taking your react components and rendering them into strings of text or HTML that can be sent as an email.
+
+Redwood currently maintains and provides the following renderers:
+* [@redwoodjs/mailer-renderer-react-email](https://github.com/redwoodjs/redwood/tree/main/packages/mailer/renderers/react-email) which is based on [https://react.email/](https://react.email/)
+* [@redwoodjs/mailer-renderer-mjml-react](https://github.com/redwoodjs/redwood/tree/main/packages/mailer/renderers/mjml-react) which is based on [mjml-react](https://github.com/Faire/mjml-react)
+
+You can also find community-maintained renderers by searching across npm, our forums, and our other community spaces.
+
+> Email clients are notoriously inconsistent in how they each render HTML into the visual email content. It is important to consider using a good react library to help you write components that will produce good-looking emails that will be rendered consistently between email clients.
```

Comment:
Instead of a quote, use admonition like 

```::: note``` or ``:: important```

Reply:
Updated. Note I used `:::warning` which currently renders red and with the danger heading. We're planning on updating to docusaurus v3 where it's been changed to render yellow and with the warning heading. I think we can leave it as `:::warning` just now and let it update as a side effect just fine.